src/ports/orders/utils.ts

```diff
@@ -182,6 +202,9 @@ export function fromOrderFragment(
     expiresAt: +fragment.expiresAt,
     createdAt: +fragment.createdAt * 1000,
     updatedAt: +fragment.updatedAt * 1000,
+    issuedId: fragment.nft.issuedId
```

Comment:
I wouldn't override the `issuedId` with the `tokenId`, let's add both fields and set `undefined` for `issuedId` if there's no value. WDYT?